runtime/compiler/control/OptionsPostRestore.cpp

```diff
@@ -331,79 +253,45 @@ J9::OptionsPostRestore::iterateOverExternalOptions()
 void
 J9::OptionsPostRestore::processJitServerOptions()
    {
+#if defined(J9VM_OPT_JITSERVER)
    if (_argIndexUseJITServer >= _argIndexDisableUseJITServer)
```

Comment:
It is worth noting that this may need to be changed in the future to account for the fact that in CRIU mode, JITServer is only enabled by default in non-portable mode (https://github.com/eclipse-openj9/openj9/pull/15674)